python/core/additions/qgis.py

```diff
@@ -0,0 +1,18 @@
+from qgis.core import Qgis
+from qgis.core import geoWkt
+from qgis.core import projectScales
+from qgis.core import geoProj4
+from qgis.core import geoEpsgCrsAuthId
+from qgis.core import geoNone
+
+from qgis import core
+
+Qgis.QGIS_VERSION = Qgis.version()
+Qgis.QGIS_VERSIONi_INT = Qgis.versionInt()
+Qgis.QGIS_VERSION_RELEASE_NAME = Qgis.releaseName()
+
+core.GEOWKT = geoWkt()
+core.PROJECT_SCALES = projectScales()
```

Comment:
Side question, is this can be renamed? It's not really projectScales, it's just the default scales if it's not found first in the real project scale list and not found in the QSettings. This is useful in the QGIS Server context for instance.
I indeed needed these values last week, but I noticed it was not sipified.

Reply:
Yep, just stumbled upon them too.

What about `Qgis.DEFAULT_PROJECT_SCALES`?

Reply:
Yes, fine for me. Thanks.

Reply:
Implemented